web/src/cypress/integration/services.ts

```diff
@@ -547,22 +547,6 @@ function testServices(screen: ScreenFormat) {
       }),
     )
 
-    it('should allow creating alerts', () => {
```

Comment:
Creating alerts from the service list page must continue to work.

Reply:
That the service name is auto populated, so only the summary/details needs to be filled out. Likely why we saw that regression earlier without the test catching it